src/modules/BootSupportCheck.rb

```diff
@@ -245,6 +241,21 @@ def check_zipl_part
       end
     end
 
+    def check_activate_partition
+      # activate set or there is already activate flag
+      return true if BootCommon.globals["activate"] == "true" || Yast::Storage.GetBootPartition(Yast::BootCommon.mbrDisk)
+
+      AddNewProblem(_("Activate flag is not set by installer. If it is not set at all, some BIOSes could refuse to boot."))
+      false
+    end
+
+    def check_mbr
+      return true if BootCommon.globals["generic_mbr"] == "true" || BootCommon.globals["boot_mbr"] == "true"
+
+      AddNewProblem(_("Installer do not modify disk MBR. Unless it contain already some boot code it, disk refuse to boot."))
```

Comment:
Very hard to read and get the meaning. I guess you meant something like "The installer will not modify the MBR of the disk. Unless it already contains some boot code, the disk will not be able to boot".